examples/knative/powercli/kn-pcli-tag/README.md

```diff
@@ -33,6 +33,17 @@ Start the container image by running the following command:
 docker run -e FUNCTION_DEBUG=true -e PORT=8080 --env-file docker-test-env-variable -it --rm -p 8080:8080 <docker-username>/kn-pcli-tag:1.0
 ```
 
+In the `test` directory, edit `test-payload.json`. Locate the `Vm` section of the JSON file. Change the `Name:` property from `K8s-User-Group-Test` to the name of a test VM currently in your vCenter inventory. If you do not make this change, the function will still be invoked, but the tag operation will fail because the VM will not be found.
+
+```json
+"Vm": {
+  "Name": "K8s-User-Group-Test",
```

Comment:
Do we want to keep this name then?

Reply:
I can change it to REPLACE-ME or similar. @lamw thoughts?

Reply:
Yup, that sounds good since we need the VM name to match for local testing